src/components/cards/reusables/IconOrb.tsx

```diff
@@ -33,9 +34,11 @@ export const IconOrb = ({
   textSize = '17pt',
   textWeight = 'bold',
 }: IconOrbProps) => {
+  const { isDarkMode } = useTheme();
+
   if (loaded) {
     return (
-      <ColorModeProvider value="light">
+      <ColorModeProvider value={isDarkMode ? 'dark' : 'light'}>
```

Comment:
forcing color mode to correspond strictly w/ app theme. otherwise, it depends on the background color of the view it's nested in

Reply:
nit: wondering if this should be in the parent card component? just a thot idrc